src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -234,8 +240,8 @@ public RTPEncodingDesc(
      */
     private static void applyFrameBoundsHeuristics(FrameDesc a, FrameDesc b)
     {
-        int end = a.getEnd(), start = b.getStart();
-        if (end != -1 && start != -1)
+        int aEnd = a.getEnd(), bStart = b.getStart();
```

Comment:
gotcha...i see this was existing, but having the unit in there might help? i.e. `aEnd` -> `aLastSeqNum` or something.  just a thought